docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,199 @@
+***********************************************************************
+
+The deepchecks train test validation suite is relevant any time you have data splits
+that you wish to validate:
+whether it's for comparing distributions across different data splits
+(e.g. before training a model or when splitting data for cross-validation),
+or for comparing new data batch to previous data batches.
+Here we'll use data from the lending club dataset, to demonstrate how you can run
+the suite with only a few simple lines of code, and see which kind of insights it can find.
+
+.. code-block:: bash
+
+    # Before we start, if you don't have deepchecks installed yet,
+    # make sure to run:
+    pip install deepchecks -U --quiet #--user
+"""
+
+#%%
+# Load Data and Prepare Data
+# ====================================================
+#
+# Load Data
+# -----------
+
+
+from deepchecks.tabular import datasets
+import pandas as pd
+
+# TODO - load this from datasets...
+# data = datasets.regression.avocado.load_data(data_format='DataFrame', as_train_test=False)
+data = pd.read_csv("lc-2017-18.csv", parse_dates=['issue_d'])
+data.head(2)
+
+
+#%%
+# Define Lending Club Metadata
+# ------------------------------
+
+categorical_features = ['addr_state', 'application_type', 'home_ownership', \
+  'initial_list_status', 'purpose', 'term', 'verification_status', 'sub_grade']
+index_name = 'id'
+label = 'loan_status' # 0 is DEFAULT, 1 is OK
+datetime_name = 'issue_d'
+
+
+#%%
+# Split Data
+# -------------
+
+# Use data for from June and July for train and August for test:
+train_df = data[data['issue_d'].dt.month.isin([6, 7])]
+test_df = data[data['issue_d'].dt.month.isin([8])]
+
+
+#%%
+# Run Deepchecks for Train Test Validation
+# ===========================================
+#
+# Define a Dataset Object
+# -------------------------
+#
+# Create a deepchecks Dataset, including the relevant metadata (label, date, index, etc.).
+# Check out :class:`deepchecks.tabular.Dataset` to see all of the columns that can be declared.
+
+from deepchecks.tabular import Dataset
+
+# We explicitly state the categorical features,
+# otherwise they will be automatically inferred,
+# which may not work perfectly and is not recommended.
+
+# The label can be passed as a column name or
+# as a separate pd.Series / pd.DataFrame
+
+# all metadata attributes are optional.
+# Some checks require specific attributes and otherwise will not run.
+
+train_ds = Dataset(train_df, label=label,cat_features=categorical_features, \
+                   index_name=index_name, datetime_name=datetime_name)
+test_ds = Dataset(test_df, label=label,cat_features=categorical_features, \
+                   index_name=index_name, datetime_name=datetime_name)
+
+#%%
+
+# for convenience lets save it in a dictionary so we can reuse them for future Dataset initializations
+columns_metadata = {'cat_features' : categorical_features, 'index_name': index_name,
```

Comment:
I think you can pass the columns_metadata in the dataset creation with **columns_metadata, instead of duplicating it

Reply:
yes, exactly, that's what I wanted to demonstrate (see later on in next dataset initialization)
but in first Dataset initializations preferred to have it explicitly for easier copy-paste (and edit) for user

Reply:
Actually I think the convenient way to do it is to use the .copy() method of a deepchecks Dataset. later on you can do new_test_ds = test_ds.copy(new_test_df) to create a new dataset with the same attributes as the original.

Reply:
yes, deepchecks-functionality-wise you're correct
but I felt that for quickstart demonstration it's more straightforward to use well-known python style + I want to encourage the usage/idea of "this is the metadata, and using this method will enable you clean and readable code".
Also if later on they want to initialize a new Dataset (let's say they dropped a few categorical columns), they can update a bit the dictionary, making it also more robust then using the "copy".